examples/glides-events/operator_signed_in.v1.json

```diff
@@ -1,6 +1,6 @@
 {
   "type": "com.mbta.ctd.glides.operator_signed_in.v1",
-  "specversion": "1.0",
+  "specversion": "1.1",
```

Comment:
suggestion!: `specversion` is the version of the CloudEvents specificiation, and it MUST be `1.0`: https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#specversion

Reply:
For reference, the way to bump the event version is to make a new event like `com.mbta.ctd.glides.operator_signed_in.v2`, then we publish both events until all consumers have migrated, then we can stop publishing the `.v1` event ([docs](https://github.com/mbta/schemas/blob/47ce0baae62cbea14871fc4014830861e32e19e8/docs/events/glides/index.mdx#compatibility)). (but I don't think we need to do that here.)